fuzz/fuzz_targets/bitcoin/deserialize_witness.rs

```diff
@@ -8,7 +8,7 @@ fn do_test(data: &[u8]) {
     let w: Result<Witness, _> = deserialize(data);
     if let Ok(witness) = w {
         let serialized = serialize(&witness);
-        assert_eq!(data, serialized);
+        assert_eq!(data, &serialized[..]);
```

Comment:
Do we really need this if we're bumping MSRV to 1.48 soon anyway?

Reply:
Good question. No, we do not. I'll remove it and wait for #1729.